doc/devel/README.md

```diff
@@ -0,0 +1,15 @@
+OS-Migrate Developer Documentation
+==================================
+
+* [Development environment setup](dev-env-setup.md) - environment
+  setup so that you can run tests your patches locally. Emphasizes use
```

Comment:
so you can run your patches locally maybe?

Reply:
Yea there's another spot where i also made a non-sentence :). This happens to me often because i go back and edit a word or a phrase but don't read the whole thing, sometimes when being interrupted etc. :)

Reply:
This part was already merged in another PR but i can edit it.